libs/external-db-postgres/src/postgres_utils.ts

```diff
@@ -1,9 +1,64 @@
 
 // Ported from PostgreSQL 9.2.4 source code in src/interfaces/libpq/fe-exec.c
+import { ColumnCapabilities } from '@wix-velo/velo-external-db-types'
+import { AdapterOperators } from '@wix-velo/velo-external-db-commons'
+const { eq, ne, string_contains, string_begins, string_ends, gt, gte, lt, lte, include } = AdapterOperators
+
 export const escapeIdentifier = (str: string) => str === '*' ? '*' : `"${(str || '').replace(/"/g, '""')}"`
 
 export const prepareStatementVariables = (n: number) => {
     return Array.from({ length: n }, (_, i) => i + 1)
         .map(i => `$${i}`)
         .join(', ')
 }
+
+export const prepareStatementVariablesForBulkInsert = (rowsCount: number, columnsCount: number) => {
+    const segments = []
+    for(let row=0; row < rowsCount; row++) {
+        const segment = []
+        for(let col=0; col < columnsCount; col++) {
+            segment.push(`$${col+1 + row * columnsCount}`)
+        }
+        segments.push('(' + segment.join(',') + ')')
+    }
+    return segments.join(',')
+}
```

Comment:
Would be great if you'll create `postgres.utils.spec.ts` only to test this function - just for better understanding of it because it's not so clear at the moment. (you can watch similar on `mongo_utils.spec` / `mysql_utils.spec`)